Subnets.tf

```diff
@@ -0,0 +1,110 @@
+
+resource "aws_subnet" "Geschäftsführung" {
```

Comment:
I would suggest to use lower case and english names for your resources. Also ä could lead to some problems on other systems

Maybe something like
```
resource "aws_subnet" "managment" {
```

Reply:
I have change all names and the IP-Address Range